qemu/tests/cfg/pci_bridge.cfg

```diff
@@ -1,19 +1,46 @@
 - pci_bridge: install setup image_copy unattended_install.cdrom
     virt_test_type = qemu
     login_timeout = 360
+    type = pci_bridge
+    only i440fx pseries
+    no usb2
+    start_vm = no
+    bootindex_image1 = 0
+    data_image_size = 1G
+    nics = nic1
+    nic_model_nic1 = virtio
+    device_id_nic1 = nic1
+    Linux:
+        sub_test_type = dd_test
+        dd_bs = 1K
+        dd_count = 1000
+    Windows:
+        iozone_cmd = "WIN_UTILS:\Iozone\iozone.exe -azR -r 64k -n 125M -g 512M -M -i 0"
+        iozone_cmd += " -i 1 -f %s:\testfile"
     variants:
+        - one_bridge:
+            pci_bridge_num = 1
+        - multi_parallel_bridges:
+            no block_stress
+            pci_bridge_num = 8
+    variants:
```

Comment:
We can change this line to something like `variants ***:`, such as `variants operation:`.  Then another two params `block_stress = yes`  and `need_hotplug_unplug = yes` can be removed.
In the script:
```
operation = params["operation"] 
if operation == 'block_stress':
    ***
if operation == 'hotplug_unplug':
   ***
```

Reply:
Another param `need_migrate = yes` can use this method too. remove it, and use 
```
if operation == 'need_migrate = yes':
   ***
```

Reply:
Updated, thanks!